src/redux/books/books.js

```diff
@@ -0,0 +1,36 @@
+const addbook = 'bookStore/books/addbook';
+const removebook = 'bookStore/books/removebook';
+
+const initialbooks = {
+  books: [],
+};
+
```

Comment:
- [ ] Great job working on this milestone. As per project requirements, you are supposed to make use of/define `slice` for the redux state. At the moment, you have not used slices. Even the import of `createSlice ` from `"@reduxjs/toolkit"` you don't have it.
You can have something similar to below snippet before the reducers for adding and removing books
```javascript
import { createSlice } from "@reduxjs/toolkit"

export const booksSlice = createSlice({
  name: "books",
  initialState: {
    books: [],
  },
    .
    .
    .
```
![image](https://github.com/shihabrafio/BookStore/assets/62849872/3f5cb235-b583-4f53-86ef-1731462ed7d0)